validate_maintainers.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python3
+"""
+Validate maintainer JSON files against the schema.
+
+Usage:
+    python validate_maintainers.py [--fix] [file.json ...]
+    
+Options:
+    --fix    Auto-fix common issues (case sensitivity, known aliases)
+    
+If no files are provided, validates all JSON files in content/maintainers/
+"""
+
+import json
+import sys
+from pathlib import Path
+
+try:
+    import jsonschema
+except ImportError:
+    print("Error: jsonschema not installed. Run: pip install jsonschema")
+    sys.exit(1)
+
+
+# Mapping of incorrect/variant labels to correct ones
+LABEL_FIXES = {
+    # Case sensitivity fixes
+    "github": "GitHub",
+    "Github": "GitHub",
+    "gitlab": "GitLab",
+    "linkedin": "LinkedIn",
+    "Linkedin": "LinkedIn",
+    "bluesky": "BlueSky",
+    "bitbucket": "BitBucket",
```

Comment:
```suggestion
    "bitbucket": "Bitbucket",
```

The canonical name is "Bitbucket", not "BitBucket": https://en.wikipedia.org/wiki/Bitbucket. This needs to be changed elsewhere as well.